src/features/surveys/components/SubmissionChartCard.tsx

```diff
@@ -0,0 +1,162 @@
+import { FC } from 'react';
+import { FormattedDate } from 'react-intl';
+import { linearGradientDef } from '@nivo/core';
+import { ResponsiveLine } from '@nivo/line';
+import { Box, Paper, Typography, useTheme } from '@mui/material';
+
+import SurveyDataModel from '../models/SurveyDataModel';
+import useModel from 'core/useModel';
+import ZUICard from 'zui/ZUICard';
+import ZUIFuture from 'zui/ZUIFuture';
+import ZUINumberChip from 'zui/ZUINumberChip';
+import { Msg, useMessages } from 'core/i18n';
+
+import messageIds from '../l10n/messageIds';
+
+type SubmissionChartCardProps = {
+  orgId: number;
+  surveyId: number;
+};
+
+const SubmissionChartCard: FC<SubmissionChartCardProps> = ({
+  orgId,
+  surveyId,
+}) => {
+  const theme = useTheme();
+  const messages = useMessages(messageIds);
+  const model = useModel((env) => new SurveyDataModel(env, orgId, surveyId));
+
+  return (
+    <ZUIFuture future={model.getStats()}>
+      {(data) => (
+        <ZUICard
+          header={messages.chart.header()}
+          status={
+            !!data.submissionCount && (
+              <ZUINumberChip
+                color={theme.palette.grey[200]}
+                value={data.submissionCount}
+              />
+            )
+          }
+          subheader={messages.chart.subheader({
+            days: data.submissionsByDay.length,
+          })}
```

Comment:
I think leave out the subheader if the card is in its "no data" state.

Reply:
Good idea! Will do.